### PR TITLE
Resolution docstring

### PR DESCRIPTION
**Pull request by chalmerlowe**
- [ ] closes #xxxx
- [ ] tests added / passed
- [X] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

```
################################################################################
################### Docstring (pandas.Timedelta.resolution)  ###################
################################################################################

Return a string representing the lowest (i.e. smallest) time resolution.

Each timedelta has a defined resolution that represents the lowest OR
most granular level of precision. Each level of resolution is
represented by a short string as defined below:

============ ============
Resolution   Return value
============ ============
Days         ``'D'``
Hours        ``'H'``
Minutes      ``'T'``
Seconds      ``'S'``
Milliseconds ``'L'``
Microseconds ``'U'``
Nanoseconds  ``'N'``
============ ============

Returns
-------
str
    Time resolution.

Examples
--------
**Using string input**

>>> td = pd.Timedelta('1 days 2 min 3 us 42 ns')
>>> td.resolution
'N'

>>> td = pd.Timedelta('1 days 2 min 3 us')
>>> td.resolution
'U'

>>> td = pd.Timedelta('2 min 3 s')
>>> td.resolution
'S'

**Using integer input**

>>> td = pd.Timedelta(36, unit='us')
>>> td.resolution
'U'

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	See Also section not found
```

### CODE REVIEW

**Review by WillAyd — CHANGES_REQUESTED** [review]
Thanks for the update! Certainly an improvement - just a minor comment from me but otherwise lgtm

**Comment by chalmerlowe** [comment]
@WillAyd 

I am new to contributing to `pandas`.

* What specific changes do you want me to make? I have been looking at this thread and clicking through to various links and I am not sure what changes I should make.

Are your requested changes to that I should tweak the PR to use the three commits marked above with the red `x`s?

IF yes... I would request some guidance on the matter of the Return values...

When I look at the documentation guide produced for the recent Pandas Documentation Sprints, this guidance is present on how to handle single and multiple return values:
```
**With a single return value**
Returns
-------
float
    Random number generated.

**With multiple return values**
Returns
-------
length : int
    Length of the returned string.
letters : str
    String of random letters.
```

Lastly... 
* I see that ci/circleci failed. When I visited the details, the failure does not appear to be anything related to my change (maybe I am missing something)? Any guidance you can offer is greatly appreciated.

**Comment by WillAyd** [comment]
Not sure what happened to my comment so probably an error on my end, but I was just asking why you had the table and the information directly below it as bullet points. It's the same information (?) so if it's duplicated I would just stick with the table and delete the bullets

Took a quick glance at the CI failures and they are LINT related to your change, so you will want to read through them and fix. The steps to reproduce are below:

https://pandas.pydata.org/pandas-docs/stable/contributing.html#python-pep8

That said, if you have run that locally you probably aren't seeing any errors because the diff in that command is only run against "*.py" files and not "*.pyx" which you've modified (most users only modify the former). If feeling ambitious that could be another change to update the docs to make sure that that rule covers both `.py` and `.pyx` files.

Hope that helps but if you get stuck let me know

**Comment by chalmerlowe** [comment]
Thanks. This helps.
For some reason, I thought I deleted bullets. Sigh. Will do so.
I will go back and rerun the linter checks against the .pyx

**Comment by chalmerlowe** [comment]
Fixed the linting problems.
Removed the bulleted items in deference to using the table.

Hope this works!

**Review by WillAyd — CHANGES_REQUESTED** [review]
Hmm well I am nitpicking but can we get rid of the double backquotes on the return values? Those are typically reserved for literals / code blocks which these technically are not

**Review by WillAyd — APPROVED** [review]

**Comment by chalmerlowe** [comment]
Removed the backticks. Let me know if we need any additional changes.

```################################################################################
################### Docstring (pandas.Timedelta.resolution)  ###################
################################################################################

Return a string representing the lowest (i.e. smallest) time
resolution.

Each timedelta has a defined resolution that represents the lowest OR
most granular level of precision. Each level of resolution is
represented by a short string as defined below:

============ ============
Resolution   Return value
============ ============
Days         'D'
Hours        'H'
Minutes      'T'
Seconds      'S'
Milliseconds 'L'
Microseconds 'U'
Nanoseconds  'N'
============ ============

Returns
-------
str
    Time resolution.

Examples
--------
**Using string input**

>>> td = pd.Timedelta('1 days 2 min 3 us 42 ns')
>>> td.resolution
'N'

>>> td = pd.Timedelta('1 days 2 min 3 us')
>>> td.resolution
'U'

>>> td = pd.Timedelta('2 min 3 s')
>>> td.resolution
'S'

**Using integer input**

>>> td = pd.Timedelta(36, unit='us')
>>> td.resolution
'U'

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	See Also section not found
```

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 1]
```diff
@@ -770,7 +770,53 @@ cdef class _Timedelta(timedelta):
...
-        """ return a string representing the lowest resolution that we have """
+        """
+        Return a string representing the lowest (i.e. smallest) time
+        resolution.
+
+        Each timedelta has a defined resolution that represents the lowest OR
+        most granular level of precision. Each level of resolution is
+        represented by a short string as defined below:
+
+        ============ ============
```
Comment (jreback) [inline pandas/_libs/tslibs/timedeltas.pyx]: I don't think this will render in a doc-string, not really sure what the standard is for a table like list

**Review by jreback — CHANGES_REQUESTED** [review]
cc @TomAugspurger

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 2]
```diff
@@ -770,7 +770,53 @@ cdef class _Timedelta(timedelta):
...
+        Seconds      'S'
+        Milliseconds 'L'
+        Microseconds 'U'
+        Nanoseconds  'N'
+        ============ ============
+
+        Returns
+        -------
+        str
+            Time resolution.
```
Comment (jreback) [inline pandas/_libs/tslibs/timedeltas.pyx]: Time -> Timedelta

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 3]
```diff
@@ -770,7 +770,53 @@ cdef class _Timedelta(timedelta):
...
+
+        Examples
+        --------
+        **Using string input**
```
Comment (jreback) [inline pandas/_libs/tslibs/timedeltas.pyx]: you don't need the 'Using string input' or numeric input section

File: pandas/_libs/tslibs/timedeltas.pyx [hunk 4]
```diff
@@ -770,7 +770,53 @@ cdef class _Timedelta(timedelta):
 
     @property
     def resolution(self):
-        """ return a string representing the lowest resolution that we have """
+        """
+        Return a string representing the lowest (i.e. smallest) time
```
Comment (jreback) [inline pandas/_libs/tslibs/timedeltas.pyx]: this should be a single line

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/_libs/tslibs/timedeltas.pyx (hunk 1 above)
Comment (chalmerlowe) [reply pandas/_libs/tslibs/timedeltas.pyx]: Do you have a suggestion?
Do  bulleted lists work?
```
          - Days:         'D'	
          - Hours:        'H'	
          - Minutes:      'T'	
          - Seconds:      'S'	
          - Milliseconds: 'L'	
          - Microseconds: 'U'	
          - Nanoseconds:  'N'
```

Comment (jreback) [reply pandas/_libs/tslibs/timedeltas.pyx]: yep that's prob ok

**Comment by jreback** [comment]
can you update for comments

**Comment by chalmerlowe** [comment]
I will update the PR.

**Comment by jreback** [comment]
can you update

**Comment by chalmerlowe** [comment]
@jreback not deliberately ignoring this. will get this done, so this open item can be cleared off the books.

**Comment by chalmerlowe** [comment]
Attempted to resolve the comments above.

```
################################################################################
################### Docstring (pandas.Timedelta.resolution)  ###################
################################################################################

Return a string representing the lowest timedelta resolution.

Each timedelta has a defined resolution that represents the lowest OR
most granular level of precision. Each level of resolution is
represented by a short string as defined below:

Resolution:     Return value

* Days:         'D'
* Hours:        'H'
* Minutes:      'T'
* Seconds:      'S'
* Milliseconds: 'L'
* Microseconds: 'U'
* Nanoseconds:  'N'

Returns
-------
str
    Timedelta resolution.

Examples
--------
>>> td = pd.Timedelta('1 days 2 min 3 us 42 ns')
>>> td.resolution
'N'

>>> td = pd.Timedelta('1 days 2 min 3 us')
>>> td.resolution
'U'

>>> td = pd.Timedelta('2 min 3 s')
>>> td.resolution
'S'

>>> td = pd.Timedelta(36, unit='us')
>>> td.resolution
'U'

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	See Also section not found
```

**Comment by chalmerlowe** [comment]
I tried to look in the Travis CI logs and I am not sure what exactly caused the Travis build to fail. Can someone advise?

**Comment by mroeschke** [comment]
@chalmerlowe There is a linting error:

`pandas/_libs/tslibs/timedeltas.pyx:837:4: W291 trailing whitespace`

After that fix, LGTM

**Comment by jreback** [comment]
@mroeschke can you push that fix?

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @chalmerlowe and @mroeschke for the fixup!

**Pull request merged**